src/types.rs

```diff
@@ -35,7 +35,7 @@ pub type StorageKey = Bytes32;
 pub type StorageValue = Bytes32;
 
 /// Type representing a log topic.
-pub type Topic = Bytes32;
+pub type LogTopic = Bytes32;
```

Comment:
In the future I want to start using newtype notation instead of type aliasing. This allows for better control over methods and type safety.

Reply:
Just create a PR 😉